### DISCUSSION TITLE
Créer une vue qui ne soit pas destinée à l'affichage mais à être envoyée par mail.

### DISCUSSION THREAD

**Discussion by moimp3** [rawsrc/PhpEcho#27] [Q&A] [answered]
Je crois que tout est dans le titre.
J'ai essayé en créant un flux (ob_flush) sans y parvenir.

**Comment by rawsrc** [comment, marked as answer]
salut

un truc dans ce style devrait suffire : 
```php
ob_start();
echo new PhpEcho('mail.body');
$mail_html = ob_get_flush();
```
ou utiliser la notation heredoc avec `PhpEcho` : `$mail->setCode('html');` avec la petite spécificité expliquée dans le manuel concernant l'échappement des valeurs dans la notation heredoc [par ici](https://github.com/rawsrc/PhpEcho/blob/master/README_fr.md#html-au-format-heredoc)
